client/client_test.go

```diff
@@ -10853,42 +10857,44 @@ func testClientCustomGRPCOpts(t *testing.T, sb integration.Sandbox) {
 }
 
 func testRunValidExitCodes(t *testing.T, sb integration.Sandbox) {
-	requiresLinux(t)
 	c, err := New(sb.Context(), sb.Address())
 	require.NoError(t, err)
 	defer c.Close()
 
 	// no exit codes specified, equivalent to [0]
-	out := llb.Image("busybox:latest")
-	out = out.Run(llb.Shlex(`sh -c "exit 0"`)).Root()
-	out = out.Run(llb.Shlex(`sh -c "exit 1"`)).Root()
+	imgName := integration.UnixOrWindows("busybox:latest", "nanoserver:latest")
```

Comment:
curious whether leaving out the version tag defaults to latest as with `docker pull`

Reply:
@billywr -- not really a big deal, also given that we could have chosen any tag for Windows instead of `latest`...
@iankingori -- yes, the default is `:latest` if not provided for the pulling logic. This can be left, but I guess the original writers of the tests wanted it to be explicit, which doesn't hurt.